packages/orbit-components/src/Card/CardSection/index.tsx

```diff
@@ -58,26 +80,20 @@ export default function CardSection({
 
   return (
     // Needs to capture bubbled click events from the <button> below
-    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
     <div
       className={cx(
-        "duration-fast lm:border-x border-b transition-all ease-in-out",
+        "duration-fast lm:border-x relative border-b transition-all ease-in-out",
         opened && "my-200 rounded-100 shadow-level2 [&+*]:border-t",
-        onClick != null && "hover:bg-white-normal-hover cursor-pointer",
+        onClick && !expandable && "hover:bg-white-normal-hover",
+        onClick &&
+          "has-[.orbit-card-wrapper-button:focus]:focus-within:rounded-100 has-[.orbit-card-wrapper-button:focus]:focus-within:outline-blue-normal has-[.orbit-card-wrapper-button:focus]:focus-within:outline has-[.orbit-card-wrapper-button:focus]:focus-within:outline-2",
```

Comment:
Perhaps you could also add here `has-[.orbit-card-wrapper-button:focus]:focus-within:z-10`? Border of other card sections is on top of the outline 😬 

![Snímek obrazovky 2025-01-30 v 15 28 51](https://github.com/user-attachments/assets/8198886c-7eca-464d-8cee-49848bb669e5)